package.json

```diff
@@ -22,7 +22,7 @@
     "lint": "yarn eslint && yarn prettier:check",
     "test": "jest --coverage",
     "verify": "yarn lint && yarn test",
-    "build": "yarn build:esm && yarn build:cjs && yarn build:es5 && yarn build:bundle",
+    "build": "yarn build:esm && yarn build:cjs && yarn build:es5 && yarn build:bundle && sbin/copyMinified.sh",
```

Comment:
probably better to move the copying to the `build:bundle` (it only affects the bundle)

Reply:
Done.